analysis/descriptives/table2.R

```diff
@@ -15,8 +25,10 @@ source("analysis/utility.R")
 print('Load active analyses')
 
 active_analyses <- readr::read_rds("lib/active_analyses.rds") 
-# filter(!outcome %in% c("out_date_bowel_ischaemia","out_date_intestinal_obstruction","out_date_nonalcoholic_steatohepatitis","out_date_variceal_gi_bleeding","out_date_belching")) %>% 
-# filter(analysis %in% c("sub_covid_hospitalised","sub_covid_nonhospitalised","sub_covid_history"))
+
+# Restrit analysis to cohort_name 
+active_analyses <- active_analyses[grepl(paste0("^cohort_", cohort_name, "-?"), active_analyses$name),]
```

Comment:
I think this just needs to be:
`active_analyses <- active_analyses[active_analyses$cohort==cohort_name,]`